service/storage_service.hh

```diff
@@ -760,6 +760,11 @@ private:
 public:
     future<bool> is_cleanup_allowed(sstring keyspace);
     bool is_repair_based_node_ops_enabled(streaming::stream_reason reason);
+
+private:
+    std::unordered_set<gms::inet_address> normal_state_handled_on_boot;
```

Comment:
Let's use the usual naming scheme for private variables (prepend it with `_`).

Reply:
OK. Done.